snapshot/containerd/snapshotter.go

```diff
@@ -29,7 +29,16 @@ type nsSnapshotter struct {
 
 func (s *nsSnapshotter) Stat(ctx context.Context, key string) (ctdsnapshot.Info, error) {
 	ctx = namespaces.WithNamespace(ctx, s.ns)
-	return s.Snapshotter.Stat(ctx, key)
+	info, err := s.Snapshotter.Stat(ctx, key)
+	if err == nil {
+		if _, ok := info.Labels["labels.containerd.io/gc.root"]; !ok {
```

Comment:
Do we have a constant for this string?

Reply:
We don't atm. Not sure where to add it as it isn't snapshot specific but containerd logic.